app/models/platform_table_info.rb

```diff
@@ -0,0 +1,119 @@
+class PlatformTableInfo
+  attr_accessor :name, :rows, :rows_trend, :table_size, :indexes_size, :total_size
+
+  def initialize(table_stats)
+    @name = table_stats['name']
+    @rows = table_stats['count'].to_i
+    @rows_trend = table_stats['count_trend']
+    @table_size = table_stats['table_size'].to_i
+    @indexes_size = table_stats['indexes_size'].to_i
+    @total_size = table_stats['total_size'].to_i
+  end
+
+  def self.stats(weeks: 5)
+    # This query retrieve all the table where the column created_at is present
+    created_at_tables_sql = %{
+      SELECT
+        c.relname as name
+      FROM
+        pg_class AS c
+      INNER JOIN
+        pg_attribute AS a on a.attrelid = c.oid
+      WHERE
+        a.attname = 'created_at'
+        AND c.relkind = 'r'
+    }
+
+    starting_date = weeks.weeks.ago.at_beginning_of_week
+
+    # We want to prepare the SQL queries to count the rows in the retrieved tables
```

Comment:
I would avoid the term `prepare` in general when describing SQL queries. It's confusing since it has a specific meaning in database terminology. Prefer `build`.